app/controllers/api/v3/catalog/item_type/items_controller.rb

```diff
@@ -0,0 +1,26 @@
+class API::V3::Catalog::ItemType::ItemsController < API::V3::Catalog::ItemType::BaseController
+  include ControlsItemSorting
+
+  def index
+    @items = @item_type.items
+    @items.page(params[:page] || 1).per(params[:per] || 25)
+  end
+
+  def show
+    @item = @item_type.items.find(params[:item_id])
+  end
+
+  private
+
+  def apply_search(items)
+    return items if params[:search].blank?
+
+    items.where("LOWER(search_data_#{I18n.locale}) LIKE ?", "%#{params[:search].downcase}%")
+  end
+
+  def apply_except(items)
+    return items if params[:except].blank?
+
+    items.where("id NOT IN (#{params[:except].join(', ')})")
```

Comment:
Attention là il y a moyen de faire une injection SQL...
```suggestion
    items.where.not(id: params[:except].join(', '))
```
(non testé)

Reply:
yes